app/assets/javascripts/initializers/initializeBaseUserData.js

```diff
@@ -28,6 +28,17 @@ function initializeUserSidebar(user) {
     followedTags.length === 0
       ? 'Follow tags to improve your feed'
       : 'Other Popular Tags';
+
+  followedTags.forEach(tag => {
+    const element = document.getElementById(
+      'default-sidebar-element-' + tag.name,
+    );
+
+    if (element) {
```

Comment:
I removed this logic by accident when I ported this section of code to Preact. It's just rudimentary filtering going on here, but it removes any followed tags from the list of Other Popular Tags, by literally removing the element from the Popular tags list. This should probably be handled server-side.